test/ui-testing/amendments.js

```diff
@@ -315,17 +318,18 @@ module.exports.test = (uiTestCtx) => {
         }
 
         if (amendment.editedName) {
-          it(`should edit amendment name to ${amendment.name}${amendment.editedName}`, done => {
+          it(`should edit amendment name from ${amendment.name} to ${amendment.editedName}`, done => {
             nightmare
               .click('#clickable-edit-amendment')
               .wait('#edit-amendment-name')
               .insert('#edit-amendment-name', '')
               .insert('#edit-amendment-name', amendment.editedName)
               .click('#clickable-update-amendment')
+              .waitUntilNetworkIdle(1000)
               .wait('#amendment-info')
               .evaluate(_amendment => {
                 const amendmentName = document.querySelector('[data-test-amendment-name]').textContent;
-                if (amendmentName !== _amendment.name + _amendment.editedName) throw Error(`Expected amendment name to be ${_amendment.editedName} and found ${amendmentName}`);
+                if (amendmentName !== _amendment.editedName) throw Error(`Expected amendment name to be ${_amendment.editedName} and found ${amendmentName}`);
```

Comment:
Hmmmm, so, this is interesting. Do these tests pass fully when running with and without the `--show` parameter? Because I remember we were having issues with Nightmare not clearing inputs when told to like in the `.insert()` on L325. 

At that point, we decided to move all tests away from clearing text fields and only _appending_ to them. As part of that work, it looks like I forgot to delete L325, the `.insert('#edit-amendment-name', '')`.

If the clearing/editing test is still flaky with this (especially without `--show`), then we should probably just update it to better match something like what we do in the `agreement-crud.js` test in ui-agreements.

(Also, while looking at this, I realised the same issue is probably present in ui-licenses' `crud.js` test.)

Reply:
Ok, I changed this to appending - and also in `crud.js`